docker/api/container.py

```diff
@@ -238,7 +238,7 @@ def create_container(self, image, command=None, hostname=None, user=None,
                          memswap_limit=None, cpuset=None, host_config=None,
                          mac_address=None, labels=None, volume_driver=None,
                          stop_signal=None, networking_config=None,
-                         healthcheck=None, stop_timeout=None):
+                         healthcheck=None, stop_timeout=None, init=None):
```

Comment:
I believe this needs to be an argument in `create_host_config`, not `create_container`